File: LarkCardKit/Models/Elements/Person.cs
using System.Text.Json.Serialization;

namespace LarkCardKit.Models.Elements;

public class Person : Element
{
    public override string Tag => "person";


    [JsonPropertyName("user_id")]
    public string? UserId { get; set; }

    [JsonPropertyName("open_id")]
    public string? OpenId { get; set; }

    [JsonPropertyName("union_id")]
    public string? UnionId { get; set; }

    [JsonPropertyName("style")]
    public string? Style { get; set; }

    [JsonPropertyName("show_name")]
    public bool? ShowName { get; set; }

    [JsonPropertyName("show_avatar")]
    public bool? ShowAvatar { get; set; }

}

File: TypeScript/LarkCardKit/src/models/elements/Person.ts
import type { ElementBase } from './Element';

export interface Person extends ElementBase {
  tag: 'person';
  user_id?: string;
  open_id?: string;
  union_id?: string;
  style?: string;
  show_name?: boolean;
  show_avatar?: boolean;
}

export interface PersonList extends ElementBase {
  tag: 'person_list';
  user_ids?: string[];
  open_ids?: string[];
  union_ids?: string[];
  style?: string;
}
